packages/manager/cypress/e2e/core/helpAndSupport/close-support-ticket.spec.ts

```diff
@@ -0,0 +1,164 @@
+import { cleanUp } from 'support/util/cleanup';
+import { authenticate } from 'support/api/authentication';
+import {
+  closableMessage,
+  closeButtonText,
+} from 'support/constants/help-and-support';
+
+describe('close support tickets', () => {
+  after(() => {
+    cleanUp(['linodes']);
+  });
+
+  authenticate();
+
+  /*
+   * - Opens a Help & Support ticket with mocked ticket data.
+   * - Confirms that there is no "close ticket" button showing up for the default support ticket.
+   */
+  it('cannot close a default support ticket by customers', () => {
+    const mockTicket = supportTicketFactory.build({
+      id: randomNumber(),
+      summary: randomLabel(),
+      description: randomPhrase(),
+      severity: randomItem([1, 2, 3]),
+      status: 'new',
+    });
+
+    // Get severity label for numeric severity level.
+    // Bail out if we're unable to get a valid label -- this indicates a mismatch between the test and source.
+    const severityLabel = SEVERITY_LABEL_MAP.get(mockTicket.severity!);
+    if (!severityLabel) {
+      throw new Error(
+        `Unable to retrieve label for severity level '${mockTicket.severity}'. Is this a valid support severity level?`
+      );
+    }
+
+    mockAppendFeatureFlags({
+      supportTicketSeverity: makeFeatureFlagData(true),
+    });
+    mockGetFeatureFlagClientstream();
+    mockGetSupportTickets([mockTicket]);
+    mockGetSupportTicket(mockTicket).as('getSupportTicket');
+    mockGetSupportTicketReplies(mockTicket.id, []).as('getReplies');
+
+    cy.visitWithLogin('/support/tickets');
+
+    // Confirm that tickets are listed as expected.
+    cy.findByText(mockTicket.summary).should('be.visible').click();
+
+    cy.wait(['@getSupportTicket', '@getReplies']);
+
+    cy.url().should('endWith', `/tickets/${mockTicket.id}`);
+    cy.findByText(
+      mockTicket.status.substring(0, 1).toUpperCase() +
+        mockTicket.status.substring(1)
+    ).should('be.visible');
+    cy.findByText(`#${mockTicket.id}: ${mockTicket.summary}`).should(
+      'be.visible'
+    );
+    cy.findByText(mockTicket.description).should('be.visible');
+    cy.findByText(severityLabel).should('be.visible');
+
+    // Confirm that the support ticket is not closable by default.
+    cy.findByText(closableMessage).should('not.exist');
+  });
+
+  /*
+   * - Opens a Help & Support ticket with mocked ticket data.
+   * - Confirms that the closable support ticket can be closed by customers successfully.
+   */
+  it('can close a closable support ticket', () => {
+    const mockTicket = supportTicketFactory.build({
+      id: randomNumber(),
+      summary: randomLabel(),
+      description: randomPhrase(),
+      severity: randomItem([1, 2, 3]),
+      status: 'new',
+      closable: true,
+    });
+
+    const mockClosedTicket = supportTicketFactory.build({
+      ...mockTicket,
+    });
+    mockClosedTicket.status = 'closed';
+    mockClosedTicket.closed = 'close by customers';
```

Comment:
```suggestion
    const mockClosedTicket = supportTicketFactory.build({
      ...mockTicket,
      status: "closed",
      closed: "close by customers"
    });
```
Nit: we can just set these values when building the object via the factory.